src/controllers/user.controllers.js

```diff
@@ -37,5 +39,8 @@ export const userLogin = async (data, response) => {
   const validPass = await bcrypt.compare(data.password, user.password);
   if (!validPass) return response.status(StatusCodes.BAD_REQUEST).send('Email or password is wrong');
 
+  const token = jsonwebtoken.sign({_id:  user._id}, process.env.TOKEN_SECRET)
```

Comment:
should we add TOKEN_SECRET to the env example(but without any value)?

Reply:
TOKEN_SECRET = sekretnytoken